src/core/Alert/Alert/Alert.baseStyles.ts

```diff
@@ -8,27 +8,25 @@ export const baseStyles = (theme: SuomifiTheme) => css`
 
   & .fi-alert_icon-wrapper {
     display: flex;
-    flex: 1 0 auto;
-    justify-content: flex-end;
   }
 
-  & .fi-alert_close-button-wrapper {
-    flex: 1 0 auto;
-    display: flex;
-    flex-wrap: nowrap;
-    box-sizing: border-box;
-    margin-top: 7px;
-    margin-right: ${theme.spacing.xs};
-    margin-bottom: ${theme.spacing.insetM};
+  & .fi-alert_style-wrapper {
```

Comment:
Is it kosher to have the width here in the style wrapper or should we have a separate style wrapper and then e.g. a `main-content-wrapper` for the width?

Reply:
In my opinion the current way of implementation is okay.